configuration/forms.md

```diff
@@ -344,6 +344,17 @@ Forms can include arbitrary binary data which is submitted and included in the d
 
 To mark an element as having binary data add an extra column in the XLSForm called `instance::type` and specify `binary` in the element's row.
 
+## Custom XForm Widgets
+
+Some XForm widgets have been added or modified for use in the app:
+- **Bikram Sambat Datepicker**: Calendar widget using Bikram Sambat calendar. Used by default for appropriate languages.
+- **Countdown Timer**: Set appearance of a `note` field to `countdown-timer`
+- **Contact Selector**: Create a field of type `db:${contact_type}` (eg `db:person`, `db:clinic`) with appearance `db-object`
+- **Rapid Diagnostic Test capture**: Create a string field with appearance `mrdt-verify`. Works with [rdt-capture Android application](https://github.com/medic/rdt-capture/).
```

Comment:
Could we add the extra information needed to display the image in the reports tab?

Reply:
Ya, could use extra description of what this ism although I don't think you need to do anything more to show it in the reports tab. Do you know of something else to include?